.gitignore

```diff
@@ -5,3 +5,4 @@ lib-dist
 node_modules
 
 npm-debug.log
+README.md
```

Comment:
I'm not sure we want to add this. It'd be an annoyance if/when we update the readme.